tests/functional_tests/io-tests.bats

```diff
@@ -3,41 +3,55 @@
 # all the tests related to HackerRank compatible IO tests for DSA course
 # setup and teardown functions
 setup() {
-	echo "empty setup"
+	sed -i 's/unit_tests/io_tests/' ../../execution_nodes/extract_run.sh
+	mkdir $BATS_TMPDIR/io-tests-example
 }
 
 teardown() {
-	echo "empty teardown"
+    sed -i 's/io_tests/unit_tests/' ../../execution_nodes/extract_run.sh
+	rm -rf $BATS_TMPDIR/io-tests-example
```

Comment:
That's only because shellcheck is not able to traverse through the entire bats file. Shellcheck bails out on the first shebang line of a bats file. To be consistent in the project, it is better to have double quotes for all the command-line strings having variables.

In this PR, only the bats file need to be modified; rest of the cases, shellcheck correctly interprets.